src/main/java/net/sf/jabref/importer/fileformat/FieldContentParser.java

```diff
@@ -37,7 +34,7 @@ public FieldContentParser() {
         multiLineFields.add("abstract");
         multiLineFields.add("review");
         // the file field should not be formatted, therefore we treat it as a multi line field
-        multiLineFields.addAll(Arrays.asList(Globals.prefs.getStringArray(JabRefPreferences.NON_WRAPPABLE_FIELDS)));
+        //multiLineFields.addAll(Arrays.asList(Globals.prefs.getStringArray(JabRefPreferences.NON_WRAPPABLE_FIELDS)));
```

Comment:
I remember (or better: the tests reminded me): This line was commented out because there were some null point exceptions due to uninitialized preferences. Fixed this in the new commit.